// Copyright (c) 2019 Boston Dynamics, Inc.  All rights reserved.
//
// Downloading, reproducing, distributing or otherwise using the SDK Software
// is subject to the terms and conditions of the Boston Dynamics Software
// Development Kit License (20191101-BDSDK-SL).

syntax = "proto3";

package bosdyn.api.mission;

option java_outer_classname = "MissionServiceProto";

import "bosdyn/api/mission/mission.proto";

service MissionService {
    // Get the state of the mission.
    rpc GetState(GetStateRequest) returns (GetStateResponse) {}

    // Specify an answer to the question asked by the mission.
    rpc AnswerQuestion(AnswerQuestionRequest) returns (AnswerQuestionResponse) {}
}
